react-to-do-list/src/Component/board/board.jsx

```diff
@@ -1,17 +1,33 @@
 import React, { useState } from "react";
 import "./board.css";
 import Card from "../Card/Card";
+import Popup from "../Poupup/Poupup";
 
 export default function Board({}) {
   const [todos, setTodos] = useState([]);
-
+  const [isPopupOpen, setIsPopupOpen] = useState(false);
   const addTodo = (task) => {
     setTodos([...todos, { task }]);
   };
+
+  const openPopup = () => {
+    setIsPopupOpen(true);
+  };
+
+  const closePopup = () => {
+    setIsPopupOpen(false);
+  };
+
```

Comment:
make one function for both open and close and use toggle instead
setIsPopupOpen(!isPopupOpen);